test/test_main.py

```diff
@@ -41,7 +41,7 @@ def setUp(self):
     def test_main(self):
```

Comment:
maybe rename test_main to test_main_sends_email_when_transmissions_success?

Reply:
I can do that. I originally wrote the test as simply testing `main`, but this probably clarifies a bit.

Reply:
Done.